shape/src/main/java/com/example/demo/entity/Picture.java

```diff
@@ -0,0 +1,31 @@
+package com.example.demo.entity;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+public class Picture extends Shape {
+    List<Shape> shapes;
+
+    public List<Shape> getShapes() {
+        return shapes;
+    }
+
+    public void setShapes(List<Shape> shapes) {
+        this.shapes = shapes;
+    }
+
+    public Picture(char ch, List<Shape> shapes) {
+        super(ch);
+        this.shapes = shapes;
+    }
+
+    @Override
+    public void draw() {
+        Stream.generate(() -> '_').limit(20).forEach(System.out::print);
```

Comment:
it is not better than concatenation

Reply:
better make simply: System.out.println("--------------------") // 20x    ?